portal-frontend/src/app/features/public/search/public-search.component.html

```diff
@@ -1,8 +1,9 @@
 <section>
   <h1>Welcome to the ALC Public Search</h1>
   <app-warning-banner>
-    This search returns only publicly available information. Applicants and local or First Nation governments should
-    <a routerLink="/login">log in</a> to view their detailed submissions.
+    This search returns only publicly available information from submissions created as of July 2015. To view older decisions, visit the
+    <a routerLink="https://www.alc.gov.bc.ca/application-decision-search/">Applications and Decisions</a> section of the ALC website. 
```

Comment:
I think since this is an external link, it should be an `href` rather than `routerLink`.

Reply:
tre, made changes